__tests__/app.test.js

```diff
@@ -73,13 +73,21 @@ describe('getArticleById', () => {
         });
       });
   });
-  test("400: GET /api/articles/9999999 responds with bad request article doesn't exisit", () => {
+  test("400: GET /api/articles/9999999 responds with bad request article doesn't exist", () => {
     return request(app)
       .get('/api/articles/9999999')
       .expect(400)
       .then((result) => {
-        console.log(result.body);
         expect(result.body.msg).toBe("Bad request, article doesn't exist");
```

Comment:
Be careful with what you're sending back, this could be interpreted as two errors 
1. bad request
2. article doesn't exist

Reply:
cleared out the console logs, it was meant for testing, I need to get better at removing